lib/src/component/abstract_transition.dart

```diff
@@ -123,6 +123,9 @@ abstract class AbstractTransitionComponent<T extends AbstractTransitionProps, S
   /// Whether the Element returned by [getTransitionDomNode] will have a transition event.
   bool get hasTransition => true;
 
+  /// The duration that can elapse before a transition timeout occurs.
+  Duration get transitionTimeout => const Duration(seconds: 15);
```

Comment:
Yeah, the only reason I said prop is that it would be easier to override. Good point, I'm fine leaving it as a getter.